examples/with-temporal/pages/api/orders/index.ts

```diff
@@ -0,0 +1,29 @@
+import { NextApiRequest, NextApiResponse } from 'next'
+import { Connection, WorkflowClient } from '@temporalio/client'
+import { Order } from '../../../temporal/src/interfaces/workflows'
+
+export type Data = {
+  result: string
+}
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse<Data>
+) {
+  if (req.method !== 'POST') {
+    res.send({ result: 'Error code 405: use POST' })
+    return
+  }
+
+  const userId: string = req.headers.authorization // FIXME insecure 😄
```

Comment:
FIXME 😄

Reply:
I intentionally left that in, as adding an auth system seemed unnecessary/irrelevant for the example. Is there a change you'd like to see here?

Reply:
Maybe make it more clear in the README what this example does and also what it _doesn't_. E.g. adding an explicit section on

1. You need to secure this
2. And then add your db here
3. And then this, etc

So they have guidance instead of FIXME or TODO 😄

Reply:
Sounds good, added this: https://github.com/vercel/next.js/pull/28348/commits/f8940dd1373225fe6bda2fa720571602e521ac8f